pom.xml

```diff
@@ -124,6 +134,57 @@
           <version>1</version>
         </configuration>
       </plugin>
+
+      <!-- Manage npm installation and dependancies -->
+      <plugin>
+        <groupId>com.github.eirslett</groupId>
+        <artifactId>frontend-maven-plugin</artifactId>
+        <version>1.12.0</version>
+
+        <configuration>
+          <nodeVersion>${node.version}</nodeVersion>
+          <workingDirectory>src/main/react-webapp</workingDirectory>
+          <installDirectory></installDirectory>
```

Comment:
I recommend setting installDirectory to target. This follows maven convention so `mvn clean` actually returns the local directory to a clean state.